geo/CHANGES.md

```diff
@@ -29,6 +29,12 @@
 * Add `wkt!` macro to define geometries at compile time.
   <https://github.com/georust/geo/pull/1063>
 
+TODO Complete this
+* Add `triangulate_delaunay`:
+  * In order to build need to add:
```

Comment:
this can be removed now

Reply:
Looking at the above examples I suspect we will need to add the georust PR reference.  I was thinking this section would be completed as a part of the prep to upstream work

Reply:
I was referring to the build instructions which were about the fortran headache which has been removed.

However, more broadly, it isnt worth keeping this CHANGES.md entry in our branch.  It will only cause merge conflicts when our branch is rebased.  Add it on your own fork's branch when getting read to submit the PR upstream.